Rakefile

```diff
@@ -1,9 +1,20 @@
 require 'bundler/gem_tasks'
 require 'rubocop/rake_task'
+require 'github_changelog_generator/task'
 
 desc 'Run RuboCop on the lib directory'
 RuboCop::RakeTask.new(:rubocop) do |task|
   task.patterns = ['lib/**/*.rb']
 end
 
 task test: %w[clean rubocop]
+
+GitHubChangelogGenerator::RakeTask.new :changelog do |config|
+  version = Hiera::Backend::Eyaml::Encryptors::GpgVersion::VERSION
+  config.future_release = "v#{version}" if version =~ %r{^\d+\.\d+.\d+$}
+  config.header = "# Changelog\n\nAll notable changes to this project will be documented in this file."
+  config.exclude_labels = %w[duplicate question invalid wontfix wont-fix skip-changelog]
+  config.user = 'voxpupuli'
+  config.project = 'hiera-eyaml-gpg'
+  config.since_tag = 'vp_migration'
```

Comment:
I created and pushed this tag to mark the point VP took over.

Reply:
If you have a history file, that also determines a since tag I believe. Is this needed? No objections to adding it.

Reply:
It can, but I'm working around the lack of tags and old release process.  `vp_migration` is a commit that doesn't properly line up with a version in rubygems.
eg the version number in the code in git was updated to 0.6 almost 3 years after 0.6 was pushed to rubygems.  If there existed a commit that I was comfortable tagging as `v0.6` I would have done that instead.